sotodlib/core/metadata/manifest.py

```diff
@@ -828,7 +828,7 @@ def main(args=None):
         print('   ' + '-' * (len(hdr) - 3))
         for row in schema:
             if row['purpose'] == 'out':
-                count = len(db.conn.execute('select distinct "%s" from map' % row['field']).fetchall())
+                count = len(db.conn.execute('select distinct `%s` from map' % row['field']).fetchall())
```

Comment:
Can you change this to the same style as the rest?

Reply:
You mean just have the string in double-quotes? "select ..."?

Reply:
Yes, and instead of the back quote a single quote:
```
count = len(db.conn.execute("select distinct '%s' from map" % row['field']).fetchall())
```

You set the rest with double quotes apart from this one and the one in obsdb. :)

Reply:
I'll just note that I hit this issue with python-3.12 testing in obsfiledb.py, line 164.  Single quotes required in sql commands...

Reply:
@iparask  This isn't a string literal, it's field name, so backticks are appropriate I think.